### PR TITLE
Add parameters to specify maximum allowd time jump

### PR DESCRIPTION
**Pull request by at-wat**

### CODE REVIEW

File: src/param.c [hunk 1]
```diff
@@ -940,6 +940,36 @@ int set_paramptr(FILE* paramfile)
...
+  }
+  else
+  {
+    g_P[YP_PARAM_MAX_TIME_JUMP][0] = g_P[YP_PARAM_CYCLE][0];
+  }
+  if (g_P_set[YP_PARAM_MAX_TIME_JUMP_NEG][0])
+  {
+    if (g_P_set[YP_PARAM_MAX_TIME_JUMP_NEG][0] <= 0)
+    {
+      yprintf(OUTPUT_LV_ERROR, "ERROR: -MAX_TIME_JUMP must be > 0.0.\n");
```
Comment (Taka-Kazu) [inline src/param.c:959]: [optional] `MAX_TIME_JUMP` is another parameter so use the name `MAX_TIME_JUMP_NEG` might be clearer here. 

```suggestion
      yprintf(OUTPUT_LV_ERROR, "ERROR: MAX_TIME_JUMP_NEG must be > 0.0.\n");
```

**Review by Taka-Kazu — APPROVED** [review]

File: src/param.c (hunk 1 above)
Comment (at-wat) [reply src/param.c:959]: `MAX_TIME_JUMP_NEG` is internal enum name and user can only specify `-MAX_TIME_JUMP` in the parameter file.
(This format was originally introduced to support different friction parameters for each rotation direction)

Comment (Taka-Kazu) [reply src/param.c:959]: I understand now.

File: src/param.c [hunk 2]
```diff
@@ -940,6 +940,36 @@ int set_paramptr(FILE* paramfile)
     g_P_changed[YP_PARAM_TORQUE_UNIT][j] = ischanged_p(YP_PARAM_TORQUE_FINENESS, j);
   }
 
+  if (g_P_set[YP_PARAM_MAX_TIME_JUMP][0])
+  {
+    if (g_P_set[YP_PARAM_MAX_TIME_JUMP][0] <= 0)
```
Comment (at-wat) [inline src/param.c:945]:
```suggestion
    if (g_P[YP_PARAM_MAX_TIME_JUMP][0] <= 0)
```

**Pull request merged**